pkg/controllers/vrpq/query_reconciler.go

```diff
@@ -79,10 +78,32 @@ func (q *QueryReconciler) Reconcile(ctx context.Context, _ *ctrl.Request) (ctrl.
 		return res, err
 	}
 
-	// setup dispatcher for vclusterops API
-	dispatcher, err := q.makeDispatcher(q.Log, q.Vdb, nil /*password*/)
+	// check version for Vdb, the minimim version should be 24.2.0
+	vinf, err := q.Vdb.MakeVersionInfoCheck()
 	if err != nil {
-		return ctrl.Result{}, err
+		return ctrl.Result{}, nil
+	}
+	if !vinf.IsEqualOrNewer(v1.RestoreSupportedMinVersion) {
+		q.VRec.Event(q.Vrpq, corev1.EventTypeWarning, events.IncompatibleDB, "Incompatibility with the database")
+		err = vrpqstatus.Update(ctx, q.VRec.Client, q.VRec.Log, q.Vrpq,
+			[]*metav1.Condition{v1.MakeCondition(vapi.QueryComplete, metav1.ConditionTrue, "IncompatibleDB")}, stateFailedQuery, nil)
+		if err != nil {
+			return ctrl.Result{}, err
+		}
+		return ctrl.Result{}, nil
+	}
+
+	// setup dispatcher for vclusterops API
+	dispatcher := q.makeDispatcher(q.Log, q.Vdb, nil /*password*/)
+	if dispatcher == nil {
```

Comment:
I'm not a fan of the of this side affect (i.e. if dispatch cannot be made, then it must mean its admintools deployments). Lets be explicit with our check.